components/web-components/toggle/index.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+
+import * as S from './style'
+
+export type Size = 'sm' | 'lg'
+export type Brand = 'vpn' | 'shields'
```

Comment:
hmm, in the design system it seems like the "fabulously large toggle" (FLT) is the only version that has a difference between shields and vpn versions, and is quite different from the default and small variants which always have purple highlights, apart from when it's the "colored" version and it's always white highlight. The FLT always has white highlights, like the "colored" version, but has the branded gradient backrounds.

Perhaps we should split the FLT to a different export and add the Brand prop to that? That would more closely resemble the Figma library:

```ts
interface ToggleProps {
  size?: 'sm' | 'default'
  theme?: 'default' | 'colored' // default is automatically the light or dark version
}

interface FabulouslyLargeToggleProps extends ToggleProps {
  brand?: 'vpn' | 'shields'
}
```

Reply:
makes sense.